spanner/spanner_arrays/main.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample spanner_arrays is a basic program that queries Google's Cloud Spanner
```

Comment:
"Sample spanner_arrays queries Google's Cloud Spanner and returns an array within the results."

The "basic" part may be discouraging if the sample isn't immediately understood.

Reply:
Good point; thanks